gulpfile.js

```diff
@@ -52,7 +52,7 @@ const Flags =
     },
     error_sound       :
     {
-        defaultValue : false,
+        defaultValue : true,

```

Comment:
oh shoot, i should change that back. I thought it was suppose to make some kind of sound if there was an error, but it didn't seem to work so i should change it back.